changelogs/current.yaml

```diff
@@ -36,6 +36,9 @@ behavior_changes:
     Handle empty response bodies in grpc_http1_reverse_bridge. This may cause problems for clients expecting the filter to crash
     for empty responses.  This behavioral change can be temporarily reverted by setting runtime guard
     ``envoy.reloadable_features.grpc_http1_reverse_bridge_handle_empty_response`` to false.
+- area: admin
+  change: |
+    Switch no admin ``warning`` -> ``info``.
```

Comment:
This should be under `minor_behavior_changes` I think - it's not "expected to cause an incompatibility if applicable; deployment changes are likely required".

Reply:
not to envoy anyhow - agree that its minor from envoy pov

Reply:
arggh - different pr - totally agree!